packages/1ui/src/components/IdentityRow/IdentityRow.tsx

```diff
@@ -37,7 +37,7 @@ export interface IdentityRowProps extends React.HTMLAttributes<HTMLDivElement> {
   numPositions: number
   tags?: TagWithValueProps[]
   userPosition?: string
-  onStakeClick: () => void
+  onStakeClick?: () => void
```

Comment:
Ah -- this is what you mentioned before as a temporary workaround until we can ensure the user's positions are available in the Activity feeds.